server/src/db.rs

```diff
@@ -45,23 +46,30 @@ pub struct NewGameEvent {
     pub event: serde_json::Value,
 }
 
-impl<DB: Backend> FromSql<Integer, DB> for GameId
+impl<DB: Backend> FromSql<Text, DB> for GameId
 where
-    i32: FromSql<Integer, DB>,
+    String: FromSql<Text, DB>,
+    //*const str: FromSql<Text, DB>,
```

Comment:
Maybe remove the commented out bit :)

Reply:
it's part of the "alternative" alloc-free thingy below